concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-48	48	boca	mouth	674	MOUTH
+Lionnet-1985-364-49	49	bola	ball	1845	BALL
+Lionnet-1985-364-50	50	brasa	embers	67	EMBERS
+Lionnet-1985-364-51	51	brazo, mano	arm, hand	2121	ARM OR HAND
+Lionnet-1985-364-52	52	bueno	good	1035	GOOD
+Lionnet-1985-364-53	53	buho	owl	735	OWL
+Lionnet-1985-364-54	54	cabello	hair	1040	HAIR
+Lionnet-1985-364-55	55	cabeza	head	1256	HEAD
+Lionnet-1985-364-56	56	caerse	fall down	2894	TUMBLE (FALL DOWN)
+Lionnet-1985-364-57	57	caerse el cabello	lose hair		
+Lionnet-1985-364-58	58	camarón	shrimp	2349	SHRIMP
+Lionnet-1985-364-59	59	camino	path, road	2457	PATH OR ROAD
+Lionnet-1985-364-60	60	caña	cane		
+Lionnet-1985-364-61	61	cansarse	become tired	1996	GET TIRED
+Lionnet-1985-364-62	62	cantar	sing	1261	SING
+Lionnet-1985-364-63	63	carne	meat	634	MEAT
+Lionnet-1985-364-64	64	carne, cuerpo	flesh, body	2615	FLESH OR MEAT
+Lionnet-1985-364-65	65	carrizo	reeds	659	REED
+Lionnet-1985-364-66	66	casa	house	1252	HOUSE
+Lionnet-1985-364-67	67	casarse el hombre	marry (male subject)	2164	MARRY (AS MAN)
+Lionnet-1985-364-68	68	cazar	hunt	1435	HUNT
+Lionnet-1985-364-69	69	ceniza	ash	646	ASH
+Lionnet-1985-364-70	70	cerrar	shut	1046	SHUT
+Lionnet-1985-364-71	71	cerrar los ojos	close eyes	2063	CLOSE (EYES)
+Lionnet-1985-364-72	72	cerro	hill	614	HILL
+Lionnet-1985-364-73	73	cesto	basket	1539	BASKET
+Lionnet-1985-364-74	74	chapulín	grasshopper	607	GRASSHOPPER
+Lionnet-1985-364-75	75	chiflar	whistle	1025	WHISTLE
+Lionnet-1985-364-76	76	chile	chili		
+Lionnet-1985-364-77	77	cielo	sky	1732	SKY
+Lionnet-1985-364-78	78	ciempiés	centipede	1974	CENTIPEDE
+Lionnet-1985-364-79	79	cinco	five	493	FIVE
+Lionnet-1985-364-80	80	clase de gavilán	type of sparrowhawk		
+Lionnet-1985-364-81	81	clase de hormiga	type of ant		
+Lionnet-1985-364-82	82	clase de mezcal	type of agave		
+Lionnet-1985-364-83	83	coa	digging stick		
+Lionnet-1985-364-84	84	cobija	blanket	806	BLANKET
+Lionnet-1985-364-85	85	cocer al horno	bake in oven	183	BAKE
+Lionnet-1985-364-86	86	coger	pick, take	2148	PICK
+Lionnet-1985-364-87	87	cola	tail	1220	TAIL
+Lionnet-1985-364-88	88	collar	necklace	1961	NECKLACE
+Lionnet-1985-364-89	89	comer	eat	1336	EAT
+Lionnet-1985-364-90	90	concha	shell	598	SHELL
+Lionnet-1985-364-91	91	conejo	rabbit	1136	RABBIT
+Lionnet-1985-364-92	92	copete	tuft		
+Lionnet-1985-364-93	93	corazón	heart	1223	HEART
+Lionnet-1985-364-94	94	cortar	cut	1432	CUT
+Lionnet-1985-364-95	95	cortarse	cut (oneself/itself)		
+Lionnet-1985-364-96	96	coyote	coyote	912	COYOTE
+Lionnet-1985-364-97	97	crecer en edad	become older		
+Lionnet-1985-364-98	98	creer	believe	1890	BELIEVE
+Lionnet-1985-364-99	99	cría de venado	fawn		
+Lionnet-1985-364-100	100	crudo	raw	1959	RAW
+Lionnet-1985-364-101	101	cuatro	four	1500	FOUR
+Lionnet-1985-364-102	102	cuchillo	knife	1352	KNIFE
+Lionnet-1985-364-103	103	cuello	neck	1333	NECK
+Lionnet-1985-364-104	104	cuerno	horn	1393	HORN (ANATOMY)
+Lionnet-1985-364-105	105	cuerpo	body	1480	BODY
+Lionnet-1985-364-106	106	cuervo	crow	913	CROW
```

Comment:
Is "cuervo" not RAVEN? I am not 100% sure.

Reply:
I believe it can be both. Maybe RAVEN is the more prototypical "cuervo" in Spanish? Not sure either. I mapped it like this because, for what I searched, a raven is a big crow and crow is a term that involves a lot of corvus species.

Reply:
Are you sure? For what I find, both are members if Corvus, but still different species within the family.

Reply:
Ok. I'll map it as RAVEN, then.

Reply:
I kept checking, and even though raven and crows are different species, I did not pay sufficient attention to your remark on "cuervo": It can indeed refer both to the genus and to the species. Further, previous cases of "cuervo" have all been mapped to CROW, so we should keep this mapping. Sorry for the mistake!

Reply:
No prob :)